swagger_server/config.ini.template

```diff
@@ -0,0 +1,7 @@
+; Copy over this template file and rename it to config.ini
+; Replace the default configurations below with the desired values
+
+[DEFAULT]
+
+EVALUATION_TYPE=dryrun
+SCENARIO_DIRECTORY=swagger_server/itm/data/%(EVALUATION_TYPE)s/
```

Comment:
Haven't actually tried this out yet, but is that trailing `s` supposed to be there?  The folders are currently `metrics` and `dryrun`, but it looks like you're unconditionally adding a trailing `s` to the path.  As written, wouldn't this require the path be `swagger_server/itm/data/dryruns/`?

Reply:
the trailing `s` is supposed to be there, it confused me as well when I read the documentation. That is how the `.ini` files do value interpolation. https://docs.python.org/3/library/configparser.html#interpolation-of-values